pages/glossary/visual-context.md

```diff
@@ -3,12 +3,12 @@ title: Visual context
 key: visual-context
 ---
 
-The visual context of some piece of information (_e.g._ an element) is everything that is visually or logically located near it. Only [visible](#visible) information may be part of the visual context.
+The visual context of a node is everything that is visually or logically located near it when the [document](https://dom.spec.whatwg.org/#concept-document) containing it is rendered. Only [visible](#visible) information may be part of the visual context.
```

Comment:
Can you explain what it means for something to be "logically located"? That phrase doesn't make much sense to me.